pom.xml

```diff
@@ -32,53 +32,53 @@
 		<blueocean-rest-impl.version>1.0-alpha-4</blueocean-rest-impl.version>
 		<blueocean-web.version>1.0-alpha-4</blueocean-web.version>
 		<!-- Pipeline (nee Workflow) plugins -->
-		<durable-task.version>1.11</durable-task.version>
+		<durable-task.version>1.12</durable-task.version>
 		<pipeline-build-step.version>2.2</pipeline-build-step.version>
-		<pipeline-input-step.version>2.0</pipeline-input-step.version>
-		<pipeline-rest-api.version>1.6</pipeline-rest-api.version>
+		<pipeline-input-step.version>2.1</pipeline-input-step.version>
+		<pipeline-rest-api.version>1.67</pipeline-rest-api.version>
```

Comment:
This is not a valid version for this plugin. The available versions are `1.7` or `2.0` according to https://updates.jenkins-ci.org/download/plugins/pipeline-rest-api/